api/src/main/java/com/cloud/offering/DiskOffering.java

```diff
@@ -37,7 +37,7 @@ enum State {
     State getState();
 
     enum DiskCacheMode {
-        NONE("none"), WRITEBACK("writeback"), WRITETHROUGH("writethrough");
+        HYPERVISOR_DEFAULT("hypervisor_default"), NONE("none"), WRITEBACK("writeback"), WRITETHROUGH("writethrough");
```

Comment:
Although there shouldn't be any issue with changing the order of the enum. I would put the new value at the end of the order.